usr/local/share/bastille/setup.sh

```diff
@@ -30,24 +30,28 @@
 
 bastille_config="/usr/local/etc/bastille/bastille.conf"
 . /usr/local/share/bastille/common.sh
+
+if [ ! -f ${bastille_config} ]; then
+    cp /usr/local/etc/bastille/bastille.conf.sample ${bastille_config}
+fi
 . ${bastille_config}
 
 usage() {
-    error_exit "Usage: bastille setup [pf|bastille0|zfs|vnet]"
+    error_exit "Usage: bastille setup [pf|network|zfs|vnet]"
```

Comment:
I see you are trying to make a change here that is not related to the topic or title of the PR.

Anyway, after executing the `setup` command, it actually created the `bastille0` bridge, so if you want this change to be here, please check that everything has been changed including the documentation since it introduces a break change.

I'll suggest keeping this PR for the bastille_zfs_prefix thing and then opening a new PR for the networking topic as it needs more work.

Reply:
bastille0 is the default - predefined name, so I do not see issues with it. Will have a look at other points (docu).
But yes, let's remove this from the zfs topic and I will create another PR.

Reply:
or I could simply (for now and backwards compatibility) "add" trhe bastille0 option into the setup.sh

Reply:
ok, make a decision in order to be able to merge it ASAP since this zfs option is a silly issue but will produce many reports after it hits ports. If you ask me, I'll prefer another PR.

I'll like to request to @cedwards a release just for that after this is fixed.